webportal/lambda/setup_venv.sh

```diff
@@ -2,4 +2,4 @@
 
 python3 -m venv .venv
 . .venv/bin/activate
-pip3 install -r requirements.txt
\ No newline at end of file
+pip3 install -r requirements.txt -r development_requirements.txt
```

Comment:
This is neat - though the more I use pip the more I realize it is sadly lacking compared to other dependency management libraries.